spec/controllers/groups_controller_spec.rb

```diff
@@ -24,4 +24,14 @@
       end
     end
   end
+
+  describe "PUT /groups/:id" do
+    context "When group does not exist" do
```

Comment:
Will you please add another spec for when the group params are invalid?

Reply:
Done.